torch/_prims/nvfuser_prims.py

```diff
@@ -199,6 +202,47 @@ def _{fname}_nvfuser(fd, a, b, c):
     )
 
 
+def _split_dim_nvfuser(
+    fd: Any,
+    a: TensorLikeType,
+    a_shape: List[int],
```

Comment:
nitpick: ShapeType instead of List[int]?

Reply:
ShapeType is `Union[torch.Size, List[int], Tuple[int, ...]]` and here only List[int] would work because we're concatenating slices of `a_shape` to list.
Also the list is what is saved on the FX graph and passed to the executor.